test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -558,25 +558,25 @@ class test_ucp_sockaddr : public ucp_test {
         ucs_status_t status = ucp_ep_query(receiver().ep(), &attr);
         ASSERT_UCS_OK(status);
 
-        EXPECT_EQ(true, m_test_addr == attr.local_sockaddr);
+        EXPECT_TRUE(m_test_addr == attr.local_sockaddr);
```

Comment:
is it possible to do `EXPECT_EQ(m_test_addr, attr.local_sockaddr)`?
or need to add `std::iostream& operator<<(std::iostream& os, const ucs::sock_addr_storage& addr)` to make it work?

Reply:
@yosefe Changed to EXPECT_EQ